app/infrastructure/middlewares/performance.py

```
@@ -1,44 +1,75 @@
 from fastapi import Request
 from starlette.responses import Response
 from starlette.middleware.base import BaseHTTPMiddleware
+from starlette.exceptions import HTTPException
 
 from app.infrastructure.broadcaster import broadcaster
 
 logger = logging.getLogger(__name__)
 
 
 class PerformanceMiddleware(BaseHTTPMiddleware):
+    def __init__(
```

Ruff (0.14.7)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)